assessments/projects/jaeger/self-assessment.md

```diff
@@ -0,0 +1,317 @@
+# Jaeger Self-assessment
+
+
+## Table of contents
+
+* [Metadata](#metadata)
+  * [Security links](#security-links)
+* [Overview](#overview)
+  * [Actors](#actors)
+  * [Actions](#actions)
+  * [Background](#background)
+  * [Goals](#goals)
+  * [Non-goals](#non-goals)
+* [Self-assessment use](#self-assessment-use)
+* [Security functions and features](#security-functions-and-features)
+* [Project compliance](#project-compliance)
+* [Secure development practices](#secure-development-practices)
+* [Security issue resolution](#security-issue-resolution)
+* [Appendix](#appendix)
+
+## Metadata
+
+|   |  |
+| -- | -- |
+| **Software** | [A link to Jaeger’s repository.](https://github.com/jaegertracing/jaeger)  |
+| **Security Provider** | No, the main function of this project is to enable distributed tracing in an organization’s tech infrastructure. Security is not the primary objective.  |
+| **Languages** | <ul><li>Go</li><li>Shell</li><li>Makefile</li><li>Python</li><li>Dockerfile</li></ul> |
+| **SBOM** | Jaeger has a workflow script that produces an SBOM upon release to comply with CLOMonitor requirements  [here](https://github.com/jaegertracing/jaeger/pull/3987) |
+| | |
+
+### Security links
+
+| Doc | url |
+| -- | -- |
+| Security file | [SECURITY.md](https://github.com/jaegertracing/jaeger/blob/main/SECURITY.md) |
```

Comment:
The team states website information preceeds the SECURITY.md, please include this in this table as well 
Ref: https://www.jaegertracing.io/report-security-issue/